src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationImageComponent/index.js

```diff
@@ -32,11 +32,13 @@ class TCEvaluationImageComponent extends Component {
              ref={this.imgEl}
              width={this.state.width}
              height={this.state.height}
-             onLoad={this.setHeight.bind(this)} />
+             onLoad={this.setHeight.bind(this)} 
+	     alt="Can't Find" />
```

Comment:
Let's leave the alt text blank (empty string), since there isn't a good alternate explanation for the contents of the image -- I don't think "Can't Find" is quite correct, since for a user with a screen reader, the image is found, it's just that they can't see it.